src/pybind/mgr/mgr_module.py

```diff
@@ -952,7 +953,7 @@ def get_store_prefix(self, key_prefix):
         return self._ceph_get_store_prefix(key_prefix)
 
     def _set_localized(self, key, val, setter):
-        return setter(self.get_mgr_id() + '/' + key, val)
+        return setter(_get_localized_key(self.get_mgr_id(), key), val)
```

Comment:
Just realized about this, and I have to say that I find this a bit inside-out, like the following snippet:
```py
def formatter(fmt, var, func):
    return func(fmt.format(var))

formatter("hello {}", "world", print)
formatter("hello {}", "world", logger.info)
```

I understand that it's a shortcut due to the code duplication because of the Active-Standby modules.